content/en/contribute/medic/product-development-process/focused-groups.md

```diff
@@ -32,6 +32,14 @@ Team members:
 - 2 Software Developers
 - Quality Assurance Engineer
 
+Videos:
```

Comment:
@mrjones-plip Is this okay? I'm not sure if it's okay to add the drive link or if it's better to upload it to YouTube.

Reply:
Let me give you two new links, both youtube.  The video I just made about Allies is OK to be public, but the brown bag was an internal video that wasn't meant for the general public. I think if we trim off all the questions at the end, the main content of me speaking is OK to publish!

I'll be right back!